index.bs

```diff
@@ -999,6 +999,13 @@ runs these steps:
          1. Let |tail| be a [=byte sequence=] containing the last
             |oldSize| - (|writePosition| + |data|.[=byte sequence/length=]) bytes of |stream|.[=[[buffer]]=].
       1. Set |stream|.[=[[buffer]]=] to the concatenation of |head|, |data| and |tail|.
+      1. If the operations modifying |stream|.[=[[buffer]]=] in the previous steps failed
+         due to exceeding the [=storage quota=], [=/reject=] |p| with a {{QuotaExceededError}} and abort,
+         leaving |stream|.[=[[buffer]]=] unmodified.
+
+         Note: [=Storage quota=] only applies to files stored in the [=origin private file system=].
+         However this operation could still fail for other files, for example if the disk being written
+         to runs out of disk space.
```

Comment:
Should we link to the new "Filling up a users disk" section here? (and below at line 1030)

Reply:
Not sure, they are kind of separate (but related) things, I don't think linking there from here really adds much.